### ISSUE TITLE
Create contributors.md

### ISSUE THREAD

**Issue by tanya-vedi** [tanya-vedi/SmartCityFramework#2]
Create contributors.md having information for beginners and the format to create pull requests.

**Comment by pranavgoel25** [comment]
Please see #4

**Comment by phavor** [comment]
Has this been taken care of?

**Comment by tanya-vedi** [comment]
> Has this been taken care of?

Yes, @phavor. You can checkout other issues or raise new issues though.
